packages/shared/src/components/cards/v1/FeedbackCard.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { MouseEventHandler, ReactElement } from 'react';
+import {
+  Button,
+  ButtonColor,
+  ButtonSize,
+  ButtonVariant,
+} from '../../buttons/ButtonV2';
+import DownvoteIcon from '../../icons/Downvote';
+import UpvoteIcon from '../../icons/Upvote';
+import { Post, UserPostVote } from '../../../graphql/posts';
+import { usePostFeedback } from '../../../hooks';
+import { useFeature } from '../../GrowthBookProvider';
+import { feature } from '../../../lib/featureManagement';
+import CloseButton from '../../CloseButton';
+import { PostCardFooter } from './PostCardFooter';
+
+interface FeedbackCardProps {
+  post: Post;
+  onUpvoteClick: MouseEventHandler;
+  onDownvoteClick: MouseEventHandler;
+  showImage?: boolean;
+}
+
+export const FeedbackCard = ({
+  post,
+  onUpvoteClick,
+  onDownvoteClick,
+  showImage = true,
+}: FeedbackCardProps): ReactElement => {
+  const { dismissFeedback } = usePostFeedback({ post });
+  const feedbackCopy = useFeature(feature.cardFeedbackCopy);
+
+  return (
+    <div className="flex-1 space-y-4">
```

Comment:
Is it necessary to have this wrapped with an extra div?

Reply:
For this questions to have a chance to answer, maybe having and TODO in the code to point here would be helpful. We just need to make sure we clear those out in the end

Reply:
Ah, cool. Yep, after checking it again, doesn't seem to be an extra one 🚀